### PR TITLE
Fix not defined env APP_URL on new setups

### PR DESCRIPTION
**Pull request by mynameisbogdan**
### 1. Why is this change necessary?
To prevent the error `Environment variable not found: "APP_URL".` on new setups.

### 2. What does this change do, exactly?
Sets a default value of `http://localhost` for `APP_URL`

### 3. Describe each step to reproduce the issue or behaviour.
Run `bin/console system:setup`

### 4. Please link to the relevant issues (if any).
shopware/production#101

### 5. Checklist

- [x] I have written tests and verified that they fail without my change
- [x] I have squashed any insignificant commits
- [x] I have created a [changelog file](https://github.com/shopware/platform/blob/trunk/adr/2020-08-03-Implement-New-Changelog.md) with all necessary information about my changes
- [x] I have written or adjusted the documentation according to my changes
- [x] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

**Comment by J-Rahe** [comment]
Hey @mynameisbogdan, could you please sign the cla?

**Comment by mynameisbogdan** [comment]
hello @J-Rahe, ok, just did. 

Thank you.

**Comment by pweyck** [comment]
Could you change it [here](https://github.com/shopware/platform/blob/96f67bfcc13b5b09e5ba360efaf105fae42d342a/src/Core/Framework/Resources/config/packages/shopware.yaml#L3)?

File: src/Core/Framework/DependencyInjection/app.xml [hunk 1]
```diff
@@ -5,6 +5,7 @@
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <parameters>
+        <parameter key="env(APP_URL)">http://localhost</parameter>
```
Comment (pweyck) [inline src/Core/Framework/DependencyInjection/app.xml]: I've recently added a default of `''`. Could you change it [here](https://github.com/shopware/platform/blob/96f67bfcc13b5b09e5ba360efaf105fae42d342a/src/Core/Framework/Resources/config/packages/shopware.yaml#L3) instead?

Comment (mynameisbogdan) [reply src/Core/Framework/DependencyInjection/app.xml]: @pweyck the bug it's fixed with that empty string as well. I created a new PR #1859 regarding just the change of the default value to `http://localhost`.

**Comment by mynameisbogdan** [comment]
I created a new PR #1859 regarding just the change of the default value to `http://localhost`.

**Pull request closed without merging**